File: app/src/main/java/com/example/applligent/nagoriengineering/ServiceReminder.java
package com.example.applligent.nagoriengineering;

import android.app.DatePickerDialog;
import android.app.TimePickerDialog;
import android.widget.DatePicker;
import android.widget.TimePicker;

public class ServiceReminder implements TimePickerDialog.OnTimeSetListener ,DatePickerDialog.OnDateSetListener {
public String time ;
public String date ;



    @Override
    public void onTimeSet(TimePicker view, int hourOfDay, int minute) {
        time =hourOfDay+ " : " +minute;

    }

    @Override
    public void onDateSet(DatePicker view, int year, int month, int dayOfMonth) {
        date= year+"/"+month+"/"+dayOfMonth;

    }

}

File: app/src/main/java/com/example/applligent/nagoriengineering/ServiceReminder.kt
package com.example.applligent.nagoriengineering

import android.app.DatePickerDialog
import android.app.TimePickerDialog
import android.util.Log
import android.widget.DatePicker
import android.widget.TimePicker

class ServiceReminder : TimePickerDialog.OnTimeSetListener, DatePickerDialog.OnDateSetListener {
    var time: String = ""
    var date: String = ""
    var temp: String? = null


    override fun onTimeSet(view: TimePicker, hourOfDay: Int, minute: Int) {
        time = hourOfDay.toString() + " : " + minute
        temp = "abc"

        Log.d("abc", temp?.length.toString())

    }

    override fun onDateSet(view: DatePicker, year: Int, month: Int, dayOfMonth: Int) {
        date = year.toString() + "/" + month + "/" + dayOfMonth

    }

}
